deploy/build.sh

```diff
@@ -20,7 +20,7 @@
 #
 
 export SERVICE_TAG=${SERVICE_TAG:-neuro-san-demos}
-export SERVICE_VERSION=${SERVICE_VERSION:-0.0.7}
+export SERVICE_VERSION=${SERVICE_VERSION:-0.0.9}
```

Comment:
Bump the version. Trying to match the `neuro-san-demos` released version

Reply:
@donn-leaf might have some CI/CD magic for making this always consistent, if he already doesn't use the SERVICE_VERSION w/ tar already.